R/commit-version.R

```diff
@@ -63,3 +70,27 @@ check_only_staged <- function(allowed_modifications) {
   modified <- staged$file
   stopifnot(all(modified %in% allowed_modifications))
 }
+
+in_example <- function() {
+  in_pkgdown <- (Sys.getenv("IN_PKGDOWN") != "")
```

Comment:
Minor: is a shortcut clearer here?

```suggestion
  if (Sys.getenv("IN_PKGDOWN") != "") {
    return(TRUE)
  }
```